src/components/Delegation/DelegatedFromUserCard.tsx

```diff
@@ -10,53 +10,54 @@ import Scale from '../Icon/Scale'
 import { snapshotUrl } from '../../entities/Proposal/utils'
 import { DelegationResult } from '../../hooks/useDelegation'
 import DelegatedCardProfile from './DelegatedCardProfile'
-import Empty from '../Proposal/Empty'
-import './DelegatedCard.css'
+import Empty from '../Common/Empty'
+import './DelegatedFromUserCard.css'
 
 const SNAPSHOT_SPACE = process.env.GATSBY_SNAPSHOT_SPACE || '' // TODO: Move to snapshot utils file
 const EDIT_DELEGATION_URL = snapshotUrl(`#/delegate/${SNAPSHOT_SPACE}`) // TODO: Move to snapshot utils file
 
 interface Props {
```

Comment:
I think the naming convention we are using for props in the repo is 'ComponentName'+'Props'.

In this case would be `DelegatedFromUserCardProps`